app/controllers/casework/reassigns_controller.rb

```diff
@@ -1,20 +1,21 @@
 module Casework
   class ReassignsController < Casework::BaseController
+    before_action :set_crime_application
     def new
       @current_assignment = current_assignment
     end
 
-    def create
+    def create # rubocop:disable Metrics/AbcSize,
+      return require_work_stream_flash if current_user.work_streams.empty?
```

Comment:
Think these lines are identical to https://github.com/ministryofjustice/laa-review-criminal-legal-aid/pull/486/files#diff-25055c48cc370e7cd6c7a09667ccddde6f433f21207db50570d7d65bca459c1eR16

Perhaps we could move them to a concern and then call them in a before action for each?